mrp_bom_current_stock/reports/report_mrpcurrentstock.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_mrpbomstructure">
+        <t t-call="web.html_container">
+            <t t-call="web.internal_layout">
+                <div class="page">
+                    <h2>BOM Stock Report</h2>
+                     <t t-foreach="docs" t-as="o">
+                         <table class="table table-condensed">
+                            <thead>
+                                <tr>
+                                    <th>Level</th>
+                                    <th>Product</th>
+                                    <th>Quantity</th>
+                                    <th>Location</th>
+                                    <th>Qty Available (Location)</th>
+                                    <th>BOM Ref</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr style="font-weight: bold;">
+                                    <td>0</td>
+                                    <td>
+                                        <span t-field="o.product_id.default_code"/>
+                                        <span t-field="o.product_id.name"/>
+                                    </td>
+                                    <td>
+                                        <span t-field="o.product_qty"/>
```

Comment:
Although I did set the operation on mrp.bom (setting the location_id where I have quantity on hand) I don't have any quantity here :confused: